pkg/awsManager/vpcVpnManager.go

```diff
@@ -0,0 +1,542 @@
+package awsManager
+
+import (
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	clientpkg "github.com/openshift/aws-account-shredder/pkg/aws"
+)
+
+func ListVPCforDeletion(client clientpkg.Client) ([]*string, error) {
+
+	var vpcToBeDeleted []*string
+	vpcList, err := client.DescribeVpcs(&ec2.DescribeVpcsInput{})
```

Comment:
Is this a paginated request? If so, likely need to paginate through it

Reply:
resolved.